src/odemis/acq/stitching/_tiledacq.py

```diff
@@ -67,11 +67,15 @@ def __init__(self, streams, stage, area, overlap, settings_obs=None, log_path=No
         """
         self._future = future
         self._streams = streams
-
+        self._stage = stage
+        self._starting_pos = {'x': area[0], 'y': area[1]}  # left, top
```

Comment:
This is left, *bottom*, as y goes from small (bottom) to big (top). I guess if you want to start at left/top, you should use area[3] (or actually normalized_area[3]).

Reply:
I got `starting_pos` from `util.normalize_rec` This should return ltrb, right?

Reply:
To clarify, normalize_rec returns minx, miny, maxx, maxy. In physical coordinates, the Y axis goes from bottom to top. So minx is the bottom.